contracts/0.4.24/Lido.sol

```diff
@@ -103,6 +73,8 @@ contract Lido is ILido, StETH, AragonApp {
     bytes32 internal constant BEACON_BALANCE_POSITION = keccak256("lido.Lido.beaconBalance");
     /// @dev number of Lido's validators available in the Beacon state
     bytes32 internal constant BEACON_VALIDATORS_POSITION = keccak256("lido.Lido.beaconValidators");
+    /// @dev amount of Ether sended to the Staking Router contract balance
```

Comment:
Let's have more precise description: amount of Ether sended to the Staking Router might be read as a compounded value

Reply:
fixed